common/IdeaNote.md

```diff
@@ -0,0 +1,29 @@
+# 아이디어 노트
+현재 학습하고 있는 것과 무관하게 스터디를 진행하며 해보면 좋겠다는 아이디어가 생긴다면, 편히 메모해놓기.  
+각자 편한 방법대로 메모, 모임에서 상의
+
+
+## JusticeChae
+
+* sample design 1
+* sample idea 2
+
+
+### sample heading idea 1
+this is sample text
+> quote text
+
+end.
+
+----
+
+## ykhaam
+
```

Comment:
1. Project ideas (general, creative, ...)
2. Questions about the theories or concepts
3. others,, etc